_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -25,32 +25,34 @@ The rule applies to any [keyboard event][]:
 
 ## Expectation 1
 
-For each test target:
- - an [instrument][] to prevent any result of the [keyboard event][] is available; or
- - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
- - an [instrument][] is available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`.
+For each test target at least one of the following is true:
+ - (**Disable**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Focus**:) any result of the event is prevented if the [event target][] doesn't have [focus][] within a 10 minute time span of the [event firing][]; or
+ - (**Remap**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false` within a 10 minute time span of the [event firing][].
```

Comment:
Expectation 2 is now somewhat wonky and it feel like it need to be merged with expectation 1.
The separation also triggers the same problem we spotted in https://github.com/act-rules/act-rules.github.io/pull/960#discussion_r434460815